### PR TITLE
Push alert detail screens and not modal

### PR DESCRIPTION
**Pull request by shichongrui**
refs: MBL-13822
affects: Parent
release note: Better display details about alerts

test plan:
 - Go to the list of alerts for a student
 - Tap on alerts, they should all push instead of modal

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Parent/Parent/Details/AnnouncementDetailsViewController.swift [hunk 1]
```diff
@@ -111,4 +111,9 @@ class AnnouncementDetailsViewController: DiscussionTopic.DetailViewController {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationController?.isNavigationBarHidden = false
```
Comment (armstrongnate) [inline Parent/Parent/Details/AnnouncementDetailsViewController.swift]: What in the world. Why are these needed?

Comment (shichongrui) [reply Parent/Parent/Details/AnnouncementDetailsViewController.swift]: No idea. I couldn't figure that out. But without it the nav bar was hidden.

Comment (shichongrui) [reply Parent/Parent/Details/AnnouncementDetailsViewController.swift]: Resulting in no way of getting away from the screen.

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1019119&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
#### Release Note: 
Better display details about alerts

#### Affected Apps: Parent

[MBL-13822](https://instructure.atlassian.net/browse/MBL-13822)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.45% | 93.44% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>
